mxcubeweb/core/models/generic.py

```diff
@@ -11,3 +11,20 @@ class SimpleNameValue(BaseModel):
 class AppSettingsModel(BaseModel):
     mode: ModeEnum = Field(ModeEnum.OSC, description="MXCuBE mode SSX or OSC")
     version: str = Field("", description="MXCuBE version")
+    mesh_result_format: str = Field(
```

Comment:
Most of the setting from the `.env` file was moved to general application settings, configured via `beamline-config.yaml`